src/display/lv_display.c

```diff
@@ -421,38 +421,7 @@ void lv_display_set_draw_buffers(lv_display_t * disp, lv_draw_buf_t * buf1, lv_d
 }
 
 void lv_display_set_buffers(lv_display_t * disp, void * buf1, void * buf2, uint32_t buf_size,
-                            lv_display_render_mode_t render_mode)
-{
-    LV_ASSERT_MSG(buf1 != NULL, "Null buffer");
-    lv_color_format_t cf = lv_display_get_color_format(disp);
-    uint32_t w = lv_display_get_horizontal_resolution(disp);
-    uint32_t h = lv_display_get_vertical_resolution(disp);
-
-    LV_ASSERT_MSG(w != 0 && h != 0, "display resolution is 0");
-
-    /* buf1 or buf2 is not aligned according to LV_DRAW_BUF_ALIGN */
-    LV_ASSERT_FORMAT_MSG(buf1 == lv_draw_buf_align(buf1, cf), "buf1 is not aligned: %p", buf1);
-    LV_ASSERT_FORMAT_MSG(buf2 == NULL || buf2 == lv_draw_buf_align(buf2, cf), "buf2 is not aligned: %p", buf2);
-
-    uint32_t stride = lv_draw_buf_width_to_stride(w, cf);
-    if(render_mode == LV_DISPLAY_RENDER_MODE_PARTIAL) {
-        /* for partial mode, we calculate the height based on the buf_size and stride */
-        h = buf_size / stride;
-        LV_ASSERT_MSG(h != 0, "the buffer is too small");
-    }
-    else {
-        LV_ASSERT_FORMAT_MSG(stride * h <= buf_size, "%s mode requires screen sized buffer(s)",
-                             render_mode == LV_DISPLAY_RENDER_MODE_FULL ? "FULL" : "DIRECT");
-    }
-
-    lv_draw_buf_init(&disp->_static_buf1, w, h, cf, stride, buf1, buf_size);
-    lv_draw_buf_init(&disp->_static_buf2, w, h, cf, stride, buf2, buf_size);
-    lv_display_set_draw_buffers(disp, &disp->_static_buf1, buf2 ? &disp->_static_buf2 : NULL);
-    lv_display_set_render_mode(disp, render_mode);
-}
-
-void lv_display_set_buffers_with_stride(lv_display_t * disp, void * buf1, void * buf2, uint32_t buf_size,
-                                        uint32_t stride, lv_display_render_mode_t render_mode)
+                            uint32_t stride, lv_display_render_mode_t render_mode)
```

Comment:
Just adding a stride parameter is an API break that we shouldn't do in a patch version.

As described [here](https://github.com/lvgl/lvgl/pull/6947#issuecomment-2408008350). I'd rather remove the `with_stride` API and add `lv_dispay_set_frame_buffers` (with a stride parameter) is v9.3.

Reply:
Looking at the patches you are working on at https://github.com/lvgl/lvgl/pull/6947/commits the function 'lv_display_set_frame_buffers' already has the stride argument and does what `lv_display_set_buffers_with_stride` is doing.
(except for the missing render_mode argument).

Since you already have something in the works, I think we should close this PR.

Reply:
Okay, let's close it now, and figure out the best way to address it in #6947. 

To avoid API breaks and incompatibilities between v9.2 and v9.3 please also remove `lv_display_set_buffers_with_stride` from v9.2.1.